src/setup/styleguide.routes.ts

```diff
@@ -1,10 +1,21 @@
-import lDefault from '@/components/l-default';
+import { ComponentPublicInstance } from 'vue';
+import lDefault from '@/components/l-default.vue';
 
-import index from '@/styleguide/routes/index';
-import sandbox from '@/styleguide/routes/sandbox';
-import icons from '@/styleguide/routes/icons';
-import forms from '@/styleguide/routes/forms';
-import notifications from '@/styleguide/routes/notifications';
+import index from '@/styleguide/routes/index.vue';
+import sandbox from '@/styleguide/routes/sandbox.vue';
+import icons from '@/styleguide/routes/icons.vue';
+import forms from '@/styleguide/routes/forms.vue';
+import notifications from '@/styleguide/routes/notifications.vue';
+
+export interface IRoute {
+  path: string;
+  name?: string;
+  component?: ComponentPublicInstance | object;
```

Comment:
What is `ComponentPublicInstance`? Can't we use that when ever the value type could be a Vue component? e.g. in vue-bem, where the class target can be an Element or a Component?